frontend/src/metabase/public/components/widgets/DisplayOptionsPane.jsx

```diff
@@ -30,63 +34,88 @@ const DisplayOptionsPane = ({
   displayOptions,
   onChangeDisplayOptions,
   canWhitelabel,
-}) => (
-  <div className={className}>
-    <DisplayOptionSection title={t`Style`}>
-      <StyleContainer>
-        <CheckBox
-          label={t`Border`}
-          checked={displayOptions.bordered}
-          onChange={e =>
-            onChangeDisplayOptions({
-              ...displayOptions,
-              bordered: e.target.checked,
-            })
-          }
-        />
-        <CheckBox
-          label={t`Title`}
-          checked={displayOptions.titled}
-          onChange={e =>
-            onChangeDisplayOptions({
-              ...displayOptions,
-              titled: e.target.checked,
-            })
+}) => {
+  const toggleId = useUniqueId("show-download-data-button");
```

Comment:
I saw `MetabotSettingWidget` uses this approach.

https://github.com/metabase/metabase/blob/9e92eb6f200621174a43580bb475f3357eb6345f/enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabotSettingWidget/MetabotSettingWidget.tsx#L17